c2pa/claim-examples/claim_create.json

```diff
@@ -0,0 +1,98 @@
+{
+    "vendor": "starlinglab",
+    "recorder": "Starling Capture",
+    "assertions": [
+        {
+            "label": "stds.schema-org.CreativeWork",
+            "data": {
+                "@context": "https://schema.org",
+                "@type": "CreativeWork",
+                "author": [
+                    {
+                        "@type": "Person",
+                        "credential": [
+                        ],
+                        "identifier": "https://hypha.coop",
+                        "name": "Benedict Lau"
+                    }
+                ]
+            }
+        },
+        {
+            "label": "c2pa.actions",
+            "data": {
+                "actions": [
+                    {
+                        "action": "c2pa.created",
+                        "when": "2021-12-10T12:08:20.440259822+00:00"
+                    }
+                ]
+            }
+        },
+        {
+            "label": "stds.exif",
+            "data": {
+                "@context": {
+                    "exif": "http://ns.adobe.com/exif/1.0/"
+                },
+                "exif:GPSLatitude": "22/1 16/1 380219916977963/8796093022208",
+                "exif:GPSLatitudeRef": "N",
+                "exif:GPSLongitude": "114/1 10/1 4036351166030807/70368744177664",
+                "exif:GPSLongitudeRef": "E",
+                "exif:GPSTimeStamp": "2021:12:10 11:23:40 +0000"
+            }
+        },
+        {
+            "label": "stds.iptc.photo-metadata",
+            "data": {
+                "@context": {
+                    "Iptc4xmpExt": "http://iptc.org/std/Iptc4xmpExt/2008-02-29/",
+                    "dc" : "http://purl.org/dc/elements/1.1/"
+                },
+                "dc:creator": [
+                    "Benedict Lau"
+                ],
+                "dc:rights": "Copyright (C) 2021 Hypha Worker Co-operative. All Rights Reserved.",
+                "Iptc4xmpExt:LocationCreated": {
+                    "Iptc4xmpExt:CountryCode": "cn",
+                    "Iptc4xmpExt:CountryName": "中国",
+                    "Iptc4xmpExt:ProviceState": "香港 Hong Kong"
```

Comment:
I assume this is what is returned from reverse geo-code. I don't know if this is valid for IPTC?

Reply:
Also note that copyright can be specified in exif.